modules/bitgo/test/v2/unit/internal/tssUtils/common.ts

```diff
@@ -8,15 +8,15 @@ export async function nockSendTxRequest(params: {coin:string, walletId: string,
 }
 
 export async function nockSendSignatureShare(params: { walletId: string, txRequestId: string, signatureShare: unknown, signerShare?: string, tssType?: 'eddsa' | 'ecdsa'}, status = 200): Promise<nock.Scope> {
-  const transactions = getRoute(params.tssType);
```

Comment:
Because this is the [getRoute](https://github.com/BitGo/BitGoJS/blob/master/modules/bitgo/test/v2/unit/internal/tssUtils/common.ts#L45) method for tests:

```
export function getRoute(tssType: 'eddsa' | 'ecdsa' = 'eddsa'): string {
  if (tssType === 'ecdsa') {
    return '/transactions/0';
  }

  return '';
}
```
It is not taking into account if API version is full or lite
This is the [sendSignatureShare method](https://github.com/BitGo/BitGoJS/blob/master/modules/sdk-core/src/bitgo/tss/common.ts#L41), if API version is full, '/transactions/0' should be added to the URL